internal/core/services/events_forwarder/events_forwarder_service.go

```diff
@@ -190,3 +174,20 @@ func (es *EventsForwarderService) forwardPlayerEvent(
 	reportPlayerEventForwardingSuccess(event.SchedulerID)
 	return nil
 }
+
+func (es *EventsForwarderService) getScheduler(ctx context.Context, schedulerName string) (*entities.Scheduler, error) {
+	var scheduler *entities.Scheduler
+	var err error
+	scheduler, _ = es.schedulerCache.GetScheduler(ctx, schedulerName)
```

Comment:
@caiordjesus I think is important to verify if schedulerCache.GetScheduler return an error. Because you are testing if scheduler was found in cache, but we can receive an error too, like connection timeout or something like that.

Could you try:

- remove lines 179 and 180
- scheduler, err := es.schedulerCache.GetScheduler(ctx, schedulerName)
- test if returns err
- ....

Reply:
Thx!